src/renderer/components/select/select.scss

```diff
@@ -104,9 +104,7 @@ html {
 
       &--is-disabled {
         cursor: not-allowed;
-        background: none !important;
```

Comment:
Are these changes by accident here?

Reply:
I honestly don't remember. I think I changed this because there are some places where the disabled text looks weird without it (since it changes the text colour).

Reply:
Should we revert this.. feels a bit dangerous change.

Reply:
Sure I can, I don't think it is as dangerous as it feels. I'll make a new PR with this change if I come across the need again.